runtime/compiler/compiler.mk

```diff
@@ -74,6 +74,7 @@ ifeq ($(PLATFORM),arm-linux-gcc-cross)
     ifeq (default,$(origin AS))
         export AS=$(OPENJ9_CC_PREFIX)-as
     endif
+    export OBJCOPY?=bcm2708hardfp-objcopy
```

Comment:
Why doesn't this use `$(OPENJ9_CC_PREFIX)` like above and below?

Reply:
This toolchain's objcopy doesn't have the same prefix as the other tools.